src/server/services/license/license.repository.ts

```diff
@@ -183,6 +112,15 @@ const licenseRepository = {
       .where(eq(licenses.id, id))
       .execute();
   },
+  findUniqueTypes: async () => {
+    return db
+      .selectDistinct({
+        type: licenses.type,
+      })
+      .from(licenses)
+      .orderBy(licenses.type)
+      .execute();
+  },
```

Comment:
In this comment I will answer on both this and the [next question](https://github.com/aarbanas/red-cross-evidence/pull/47#discussion_r1845437119). 

I was thinking that it would be wise to allow the customer to create types on its own because we can not predict what types they will think off. That is why I've decided to ditch the enum and search for distinct types in DB. That is why I've written the description on [this issue](https://github.com/aarbanas/red-cross-evidence/issues/15#issue-2390685395) so that we try to protect as much errors as possible on new type creation.